src/styles/normalize.scss

```diff
@@ -0,0 +1,13 @@
+.page {
+  margin: 0;
+  font-family: Roboto, Arial, serif;
+}
+
+h3,
+p {
+  margin: 0;
+}
```

Comment:
If your want to reset defolts styles of tags, do this in _reset.scss.

Reply:
Agree. Not critical on such small projects, but it's better to separate these things on bigger projects.